bundle/bundle.go

```diff
@@ -43,6 +45,33 @@ type Bundle struct {
 	AutoApprove bool
 }
 
+const ExtraIncludePathsKey string = "DATABRICKS_BUNDLE_INCLUDE_PATHS"
+
+// Get extra include paths from environment variable
+func GetExtraIncludePaths() []string {
+	value, exists := os.LookupEnv(ExtraIncludePathsKey)
+	if !exists {
+		return []string{}
+	}
+	var sep = ":"
+	if runtime.GOOS == "windows" {
+		sep = ";"
+	}
+	return strings.Split(value, sep)
+}
+
+// Try loading bundle config from one of the extra include paths from the environment variable
+// If not able to load, do not return an error, just return nil
+func tryLoadFromExtraIncludePaths() *Bundle {
+	for _, extraIncludePath := range GetExtraIncludePaths() {
+		bundle := &Bundle{}
+		if err := bundle.Config.Load(extraIncludePath); err == nil {
+			return bundle
+		}
+	}
+	return nil
+}
+
```

Comment:
We can't do this as is. It would imply that the path where the config file is located is used as the bundle root.

Reply:
I tried to handle this with how it is being called. The ONLY place this function is being called is AFTER the bundle root has been found and established. But there is no bundle.yaml or databricks.yaml in that file. The bundle root could have been established with `BUNDLE_ROOT` env var.